gateway/remote_blocks_backend.go

```diff
@@ -0,0 +1,56 @@
+package gateway
+
+import (
+	"net/http"
+	"time"
+
+	"github.com/ipfs/boxo/blockservice"
+	"github.com/ipfs/boxo/exchange/offline"
+	"go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp"
+)
+
+const getBlockTimeout = time.Second * 60
```

Comment:
No need to address this now, but worth leaving a TODO:

```suggestion
// TODO: make this configurable via BlocksBackendOption
const getBlockTimeout = time.Second * 60
```